admin_guide/manage_nodes.adoc

```diff
@@ -15,115 +15,125 @@ link:../architecture/infrastructure_components/kubernetes_infrastructure.html#no
+node name:
 
-****
-`$ oc get node _<node>_`
-****
+----
+$ oc get node <node>
+----
 
-The `STATUS` column in the output of these commands indicates whether the node
-is passing the health checks performed from the master. Nodes can be shown in
-this column with the following conditions:
+The `STATUS` column in the output of these commands can show nodes with the
+following conditions:
 
-.Node Conditions
+.Node Conditions [[node-conditions]]
 [cols="3a,8a",options="header"]
 |===
 
 |Condition |Description
 
 |`Ready`
-|The node has returned `StatusOK` for the health check.
+|The node is passing the health checks performed from the master by returning
+`StatusOK`.
 
 |`NotReady`
-|The health check is not passing.
+|The node is not passing the health checks performed from the master.
+
+|`SchedulingDisabled`
+|Pods cannot be link:#marking-nodes-as-unschedulable-or-schedulable[scheduled
+for placement] on the node.
+
 |===
 
 NOTE: The `STATUS` column can also show `Unknown` for a node if the CLI cannot
 find any node condition.
 
-Use the following command to get more detailed information about a node,
-including the reason for the current condition:
-****
-`$ oc describe node _<node>_`
-****
+To get more detailed information about a specific node, including the reason for
+the current condition:
 
-.Getting a Node Description
-====
+----
+$ oc describe node <node>
+----
 
+For example:
+
+====
 [options="nowrap"]
 ----
-$ oc describe node node2.example.com
-Name:	node2.example.com
+$ oc describe node node1.example.com
+Name:			node1.example.com
+Labels:			kubernetes.io/hostname=node1.example.com
+CreationTimestamp:	Wed, 10 Jun 2015 17:22:34 +0000
 Conditions:
-Kind    Status  LastProbeTime                   LastTransitionTime              Reason                                                              Message
-Ready   None    Tue, 03 Mar 2015 22:53:22 +0000 Tue, 03 Mar 2015 21:27:50 +0000 Node health check failed: kubelet /healthz endpoint returns not ok
+  Type		Status	LastHeartbeatTime			LastTransitionTime			Reason					Message
+  Ready 	True 	Wed, 10 Jun 2015 19:56:16 +0000 	Wed, 10 Jun 2015 17:22:34 +0000 	kubelet is posting ready status
+Addresses:	127.0.0.1
+Capacity:
+ memory:	1017552Ki
+ pods:		100
+ cpu:		2
+Version:
+ Kernel Version:		3.17.4-301.fc21.x86_64
+ OS Image:			Fedora 21 (Twenty One)
+ Container Runtime Version:	docker://1.6.0
+ Kubelet Version:		v0.17.1-804-g496be63
+ Kube-Proxy Version:		v0.17.1-804-g496be63
+ExternalID:			node1.example.com
+Pods:				(2 in total)
+  docker-registry-1-9yyw5
+  router-1-maytv
+No events.
 ----
 ====
 
-== Adding a Node
+== Adding Nodes
 You can add a node to an existing OpenShift instance using the `oc create`
-command and supplying a manifest for a node object. After the new node passes
-the health checks that are performed by the master,  pods can begin to be
-scheduled for the node.
+command and supplying a specification for a node object. After the newly-created
```

Comment:
Just a suggestion:

s/...and supplying a specification for a.../...and specifying a...

I'm not sure if the object itself is a _specification_, right?